recipes/gistim/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "gistim" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gistim-{{ version }}.tar.gz
+  sha256: fdf105534fd38121b8cbec7066f0626d952370a854f7c40845030c0e3315f7b0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+  run:
+    - affine
+    - geopandas
+    - matplotlib
+    - netCDF4
+    - numba >=0.50
+    - numpy
+    - pandas
+    - python >=3.6
+    - rasterio >=1.0
+    - rioxarray
+    - scipy
+    - timml
+    - xarray >=0.15
+
+test:
+  imports:
+    - gistim
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/deltares/imod/qgis-tim
+  summary: GIS utilities for Tim(ML) Analytic Element modeling
+  license: GPL-2.0-or-later
+  license_file: LICENSE
+  doc_url: https://deltares.gitlab.io/imod/qgis-tim
+  dev_url: https://gitlab.com/deltares/imod/qgis-tim
+
+extra:
+  recipe-maintainers:
+    - Huite
+    - visr
```

Comment:
Hi @visr , could you please confirm that you would like to maintain this recipe as well?

Reply:
I confirm that I'd like to maintain this recipe.